vars/withParameterizedPipeline.groovy

```diff
@@ -84,6 +84,7 @@ def call(type, String product, String component, String environment, String subs
         pipelineType: pipelineType,
         subscription: subscription,
         aksSubscription: aksSubscriptions.aat,
+        autoStartSubscription: aksSubscriptions.sandbox,
```

Comment:
this is a hack that shouldn't be done.

you should just use the aksSubscription and you can check if the name ends with SBOX:
https://github.com/hmcts/cnp-flux-config/blob/4b81dd310e26c6bad6fe8bf2b5cbc072f25d5471/apps/jenkins/jenkins/sbox-intsvc/jenkins.yaml#L77-L78

Reply:
The aksSubscription  is being passed above as aat each time though?

Reply:
That's already confusing then as aat subscription is being set to sandbox somewhere else too? 😕  Will update though